controllers/node-delete/controller.go

```diff
@@ -6,7 +6,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	objectv1 "github.com/darkowlzz/operator-toolkit/controller/external-object-sync/v1"
+	syncv1 "github.com/darkowlzz/operator-toolkit/controller/sync/v1"
```

Comment:
Same weird import separation like in namespace-delete controller.

Reply:
Yes, will fix - thanks